Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -1033,6 +1034,41 @@ UpdateFspConfig (
       case PLATFORM_ID_ADL_P_DDR5_RVP:
         FspsConfig->Usb4CmMode = 0x1;
         break;
+      case PLATFORM_ID_ADL_PS_DDR5_RVP:
+        FspsConfig->PchLanEnable = 0x0;
```

Comment:
please add these into adlps delta file, so no need to redo here.

Reply:
Done, thanks for the review.